internal/k8s/controller.go

```diff
@@ -2160,27 +2159,38 @@ func (lbc *LoadBalancerController) createIngressEx(ing *networking.Ingress, vali
 	return ingEx
 }
 
-func (lbc *LoadBalancerController) getAppProtectLogConfAndDst(ing *networking.Ingress) (logConf *unstructured.Unstructured, logDst string, err error) {
-	logConfNsN := appprotect.ParseResourceReferenceAnnotation(ing.Namespace, ing.Annotations[configs.AppProtectLogConfAnnotation])
+func (lbc *LoadBalancerController) getAppProtectLogConfAndDst(ing *networking.Ingress) (apLog []configs.AppProtectLog, err error) {
```

Comment:
because we use `err` variable throughout the code in the method body and without any intention to return its value, I suggest dropping the names in `(apLog []configs.AppProtectLog, err error)` -> `([]configs.AppProtectLog, error)` and inside the body, simply declare `var apLog []configs.AppProtectLog` . And a small suggestion, because we have a slice, better to use plural `apLogs`.